<!DOCTYPE html><html><head><meta charset='utf-8'><meta name='viewport' content='width=device-width, initial-scale=1'><script async src='https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-8422244967817077' crossorigin=anonymous'></script><link rel='stylesheet' href='/class.css'><script src='/script.js'></script></head><body><div class='topnav'></div><div class='list'><Br/><a href='1095.html'>Hemorrhoid Permane</a><Br/><a href='1096.html'>Last of Us 2 Grief</a><Br/><a href='1097.html'>Criminal Attorney,</a><Br/><a href='1098.html'>Facebook, Social M</a><Br/><a href='1099.html'>Which SELFEGGIO fr</a><Br/><a href='1100.html'>Which SELFEGGIO fr</a><Br/><a href='1101.html'>Which SELFEGGIO fr</a><Br/><a href='1102.html'>Emergency services</a><Br/><a href='1103.html'>Video Games, XBOX,</a><Br/><a href='1104.html'>Fishing, Hunting, </a></div><div class='stats'><div class='logodiv'><a href='/'><img class='logoimg' src='/img/elephant.svg' /></a></div> <Br/><a href='1093.html'>Horoscope and Astr</a><Br/><a href='1092.html'>Vegan-based Diet b</a><Br/><a href='1091.html'>Engrish as a secon</a><Br/><a href='1090.html'>Sweepstakes and Ga</a><Br/><a href='1089.html'>Non-Profit, Religi</a><Br/><a href='1088.html'>Dating, LGBTQIA+ a</a><Br/><a href='1087.html'>Work From Home, Ho</a><Br/><a href='1086.html'>Piercings, Tattoos</a><Br/><a href='1085.html'>UFC, MMA, BJJ, Kar</a><Br/><a href='1084.html'>FTL, LTL, and Long</a></div><div class='nav'><a href='1093.html'> << </a>&nbsp;&nbsp;&nbsp;&nbsp;<a href='1095.html'> >> </a></div><div class='article'>ID Enhancement and Age Selection {#S2}
=========================================

Although NGS analysis can lead to the accurate determination of the germline sequence of human TAs ([@B19]), it remains challenging to infer their evolutionary history from germline sequences alone due to genetic instability during meiosis and somatic hypermutation during the affinity maturation of antibody repertoires ([@B10]). For TAs, affinity maturation is particularly important for their ability to recognize specific antigens. In our recently developed framework of immune repertoire sequencing ([@B20]), we showed that high-throughput sequencing and bioinformatic analysis can be used to reconstruct the complete germline genes for individual B cells, to infer the lineage history, the extent of somatic mutation, and to assign somatic mutations to B cell clonal lineages. Based on this framework, here we use NGS of the B cell repertoires from 1,056 donors in the public domain ([@B21]--[@B23]) to model the evolutionary history of TAs. In addition, to investigate the evolutionary trajectories of functional TAs, we performed an *in silico* experiment using high-throughput sequencing data to characterize B cell repertoires of antibodies specific for 12 influenza vaccines used for human immunization during the H1N1 pandemic in 2009. We then combined the sequence and structural information from B cell repertoires and human TAs to reconstruct the evolutionary history of H1N1 vaccine TAs.

B cell repertoire sequencing, the inference of germline sequence, and the reconstruction of phylogenetic history rely on inferring accurate CDR3 sequences and inferring the full-length amino acid sequences of the variable domain of TAs in the germline configuration. We identified highly similar TAs by clustering and removed possible germline gene duplication events ([@B24]) from the B cell repertoires. We then inferred CDR3 sequences and V domains for each germline sequence using IMGT HighV-Quest ([@B25]) ([Figure 1A](#F1){ref-type="fig"}).

![A pipeline to model the evolutionary history of functional antibody variable domains. **(A)** The first step in reconstructing the evolutionary history of TAs is to build a database of the germline sequences of human TAs that are homologous to the given query antibody. The germline sequences are taken from the IMGT database. The phylogenetic tree is built on the resulting germline V gene sequences. **(B)** The second step in reconstructing the evolutionary history of human TAs is to identify lineage-specific substitutions in the reconstructed evolutionary tree. The first step is to align all the V domains of interest, with gaps (green arrow) at the site of a germline replacement event. The second step is to generate a phylogenetic tree for the aligned sequences with bootstrap values. The third step is to choose one or more sequence(s) from different subtrees in the phylogenetic tree as the starting point for a substitution inference model. In the case of H1N1 influenza vaccine TAs, a total of 1096 sequences were available. The phylogenetic tree of these sequences was generated from the germline sequences of V domains (blue circle). The fourth step is to build a probabilistic substitution model to map branch lengths from a phylogenetic tree back to the phylogenetic tree to identify the lineage-specific substitutions (green arrow). Note that the green arrow point may not be located at the tips of the phylogenetic tree, which can be accounted for by using the maximum likelihood estimation of branch lengths to infer the likelihood of lineage-specific substitution with the branch length and the starting point as variables. **(C)** The next step is to identify the lineage-specific substitutions in the inferred evolutionary tree of TAs. Two possible examples of lineage-specific substitutions (blue and red edges) for the influenza vaccine TAs are shown in the left panel. The right panel shows the ancestral germline sequences of the identified lineage-specific substitutions and the locations of the point mutations that occurred in the inferred evolutionary tree.](fimmu-10-01704-g0001){#F1}

To model the evolutionary history of TAs in the absence of an existing reference or template, we first reconstructed phylogenetic trees from B cell repertoires and identified lineage-specific substitutions by inferring probabilistic models of the phylogenetic tree to predict lineage-specific substitutions. We then reconstructed a phylogenetic tree from each of the repertoires, identified lineage-specific substitutions, and inferred the most likely substitution model.

Identification of B Cell Repertoires That Contain H1N1-Specific TAs and Selection of the Germline V Gene Repertoire {#S3}
===================================================================================================================

We analyzed a total of 1,056 public B cell repertoires of na√Øve or memory B cells specific for 12 different influenza vaccines. Here, "influenza vaccine" refers to either a whole influenza vaccine or an antigen of the influenza vaccine. All but three of the B cell repertoires were generated from healthy donors and two B cell repertoires were generated from donors infected with H1N1 influenza virus. Our aim was to model the evolutionary trajectories of the TAs that are specific for these 12 influenza vaccines. Since the HA and NA proteins of the H1N1 virus remain highly conserved in all human influenza viruses (with minor differences in their amino acid sequences), we expected that any TAs that target H1N1 viruses will have to recognize H1N1 antigens. To confirm that our inference strategy is working for inferring the phylogenetic history of TAs, we first applied our method to infer the evolutionary history of TAs that are specific for influenza viruses. To select TAs that are specific for H1N1 influenza virus, we collected the most similar TAs in each of the B cell repertoires from the H1N1 influenza vaccine, as well as the HA or NA sequences of the H1N1 influenza virus. We used the HA or NA protein sequences of the H1N1 influenza virus instead of the whole H1N1 influenza virus because the HA or NA protein sequences are highly similar among all H1N1 influenza viruses. We then clustered these TAs by comparing their V genes and using a sequence identity threshold of 94%. Using this approach, we were able to identify 2,600 TAs that were specific for H1N1 influenza virus, which contained at least two TAs from each of the 12 repertoires ([Supplementary Dataset 1](#SM1){ref-type="supplementary-material"}).

We were then interested in studying the evolution of the B cell repertoire of TAs specific for the H1N1 influenza vaccine. We first extracted the HA and NA sequences of H1N1 influenza virus that are found in the 12 influenza vaccine B cell repertoires and included the TAs of interest in the corresponding B cell repertoires. The full-length HA and NA protein sequences of the H1N1 influenza virus are highly similar, and thus can be expected to share most of their antigen-binding sites ([@B26]). For this reason, it is reasonable to assume that TAs that target H1N1 influenza virus also target HA. We then clustered the resulting HA sequences from each repertoire using a sequence identity threshold of 93% to identify sequence clusters that likely represent related HA sequences from a single H1N1 strain, and used only the HA sequences belonging to the same sequence clusters as the query sequences for phylogenetic tree inference. The HA sequences from each of the 12 H1N1 influenza virus are clustered into 24 sequence clusters. We observed similar clustering results with the NA sequences, except that the NA sequence clusters were larger (34 clusters for HA and 19 clusters for NA). This is likely due to the fact that the 12 influenza vaccine B cell repertoires were either produced by B cells from donors infected with the H1N1 virus or represent sequences from memory B cells ([@B21]--[@B23]), which might generate more complex and heterogeneous clonal lineages than that of the naive repertoire. In addition, we observed a lower sequence identity between the HA and NA sequences from the same strain, which suggests that the HA and NA sequences from the same strain were generated independently. For example, in donor B1H, we found that the sequence identity between the HA sequences of H1N1/2009 and H1N1/HK in the repertoire B1H\_-3 was 74.0%, while the sequence identity between the HA and NA sequences of H1N1/2009 in the repertoire B1H\_-3 was 77.0%. In this case, the HA and NA sequences belonged to different sequence clusters, indicating that both HA and NA sequences were independently generated from H1N1/2009 virus. This is also consistent with the fact that H1N1/2009 belongs to the H1N1 subtype and that the HA and NA sequences of the H1N1/2009 virus are very similar to H1N1/HK. Overall, phylogenetic trees were built for 2,600 TAs from 12 B cell repertoires that contained the TAs of interest ([Figure 2A](#F2){ref-type="fig"}). We further filtered the phylogenetic trees by selecting the B cell repertoire that contained the TAs with the highest sequence identity. As a result, we were able to select 300 TAs from B cell repertoires of seven different donors, which represented the TAs with the highest sequence identity for a given strain. The detailed sequence identity distribution is shown in [Supplementary Figure 1](#SM1){ref-type="supplementary-material"}.

![Identification of TAs with high sequence identities and evolutionary trajectories of functional influenza-specific TAs. **(A)** A total of 300 TAs from seven different H1N1-specific B cell repertoires were extracted for phylogenetic analysis. We filtered the phylogenetic tree and removed branches that did not contain any TAs to avoid inferring the evolutionary trajectories of non-functional</div></body></html><!-- 2022-06-12 21:02:26 