<!DOCTYPE html><html><head><meta charset='utf-8'><meta name='viewport' content='width=device-width, initial-scale=1'><script async src='https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-8422244967817077' crossorigin=anonymous'></script><link rel='stylesheet' href='/class.css'><script src='/script.js'></script></head><body><div class='topnav'></div><div class='list'><Br/><a href='3653.html'>Girls Gone Wilder</a><Br/><a href='3654.html'>aincel.com</a><Br/><a href='3655.html'>The Gods Are Angry</a><Br/><a href='3656.html'>Adult MP3, 18+</a><Br/><a href='3657.html'>Nude Beach Satelli</a><Br/><a href='3658.html'>YOLO, Let's Play!</a><Br/><a href='3659.html'>Old and New Bonds</a><Br/><a href='3660.html'>This isn’t who I a</a><Br/><a href='3661.html'>We Made It to the </a><Br/><a href='3662.html'>Many of the apocal</a></div><div class='stats'><div class='logodiv'><a href='/'><img class='logoimg' src='/img/elephant.svg' /></a></div> <Br/><a href='3651.html'>Over the long term</a><Br/><a href='3650.html'>Job Search, Dice, </a><Br/><a href='3649.html'>Are We Gonna Live </a><Br/><a href='3648.html'>That's Love, Baby!</a><Br/><a href='3647.html'>There's comfort in</a><Br/><a href='3646.html'>Momma didn't raise</a><Br/><a href='3645.html'>Blackmail or Betra</a><Br/><a href='3644.html'>What About Me?</a><Br/><a href='3643.html'>Shocking! Simply S</a><Br/><a href='3642.html'>If It Smells Like </a></div><div class='nav'><a href='3651.html'> << </a>&nbsp;&nbsp;&nbsp;&nbsp;<a href='3653.html'> >> </a></div><div class='article'>ID Enhancement and Age Selection {#sec2.4}
---------------------------------------

When performing TIDE, we use a window size of 200 bp. It is not necessary to choose a large window size, as the results do not change drastically when increasing the window size from 100 to 1000 bp (data not shown). We set the threshold to 1% of reads with mutations in the window (default parameters) in the pre-filtering step. The parameter "callThreshold" of the *tidecall* package can be changed to adjust the threshold when running TIDE. To remove variants with low allelic fractions (AF), we set a parameter 'threshold_max_allele_fraction' (default: 0.5), where AF can be calculated as $$AF = \frac{\#\text{total\ reads\ covering\ the\ variant}}{\#\text{total\ reads\ covering\ the\ variant} + \#\text{total\ reads\ without\ a\ variant}}$$

As a next step, we used the *bedtools closest* utility (<https://bedtools.readthedocs.io/en/latest/content/tools/closest.html>) to determine if the variants from the VAFs files have been called by other tools. We further used vcftools to look for the genomic position of the variant, i.e., the chromosomal position (BED or GFF file) of the site. We discarded sites outside exons or on chromosome Y for men and not in exons for women. Moreover, sites found with a VAF \< 0.4% were discarded. Finally, we excluded indels from further analysis. The VCF file containing all these sites was used as input for the *bamUtilities filterUnusedAlleles* (<https://genome.sph.umich.edu/wiki/BamUtil>) with parameter --b --v --f and without the parameter --q, resulting in the final called mutations. To apply IGVtools for visualization of the WGS data with mutation, we used *tiffcp_ngs_tools.py* (<https://genome.sph.umich.edu/wiki/IsoMut_Tiffcp>) to convert the tabular BAM output from TIDE to a TIF file.

2.5. DNA Isolation, WES, and Analysis of Mutations {#sec2.5}
--------------------------------------------------

DNA isolation was performed using QIAamp DNA Mini Kit (Qiagen, Hilden, Germany), according to the manufacturer\'s instructions. WES and subsequent whole-genome SNP-array analysis were performed using Human Core Exome v1.0 Beadchip (Illumina Inc., San Diego, CA). Copy number aberration analysis was performed by OncoScan® CNV Assay (Affymetrix Inc., Santa Clara, CA) according to manufacturer\'s protocol. In addition to this, WES was performed in the original patient who received radiotherapy (before the second relapse) using SureSelect Human All Exon V6 (Agilent, Santa Clara, CA). Sequence alignment and variant analysis were done using Torrent Suite V5.4.0 and variant calling was performed using Oncotagger® V4.2.1 (Agilent, Santa Clara, CA). All detected variants were annotated with IGVtools (<https://www.broadinstitute.org/igv/>). Copy number was plotted using DNAcopy R-package \[[@B20]\].

2.6. Gene Expression Microarray Analysis {#sec2.6}
----------------------------------------

RNA was extracted using a high-purity total RNA extraction kit (E.Z.N.A, Omega Bio-Tek, Norcross, GA) according to the manufacturer\'s instructions. RNA integrity was checked on an Agilent Bioanalyzer 2100 (Agilent, Santa Clara, CA). A total of 1 *μ*g RNA from the healthy volunteer\'s control sample and the patient before radiotherapy was reverse transcribed into double-stranded complementary DNA (cDNA) and transcribed into complementary RNA (cRNA) using an Affymetrix GeneChip 3′ IVT Express Kit (Affymetrix Inc., Santa Clara, CA). Hybridization was performed for 16 hours at 45°C on a Human Gene 1.0 ST Array (Affymetrix Inc., Santa Clara, CA). Data were analyzed using the RMA background correction algorithm (RMA, affy package, R version 3.2.3). All expression levels of each gene were normalized to a mean of 20,000. Differentially expressed genes were filtered using a cutoff value of *p* \< 0.05.

2.7. Statistical Analysis {#sec2.7}
-------------------------

Statistical analysis was performed using GraphPad Prism 5.0, R version 3.2.3, and IBM SPSS Statistics 24. Correlation of somatic variants and patient outcome was performed with Kaplan-Meier method. In the WES, analysis of gene expression between a healthy volunteer and the patient at relapse was performed using one-way ANOVA.

3. Results {#sec3}
==========

3.1. Discovery of Germline Heterozygous Variants in a Patient with Hematological Malignancy {#sec3.1}
-------------------------------------------------------------------------------------------

We identified three heterozygous germline variants in genes known to play an important role in chromosomal segregation in the patient with relapsed B-ALL. One was an inactivating homozygous splice site variant in *MCPH1* (NM_014752.3: c.2474+1G\>T) ([Table 1](#tab1){ref-type="table"}). This splice site variant causes the use of a new cryptic splice site and results in a premature stop codon \[[@B21]\]. One heterozygous germline variant was identified in the *BRCA1* gene (NM_007300.3: c.5560T\>A) and one heterozygous germline variant was identified in the *STK11* gene (NM_000455.4: c.A2286G). *BRCA1* (OMIM: 113705) is a tumor suppressor gene that functions as a genome guardian for genome integrity and stability, allowing cell growth. Inactivation of *BRCA1* can lead to an increase of genomic instability, chromosomal aberrations, and an increased number of sister chromatid exchanges, chromosomal breaks, and sister chromatid exchanges \[[@B22]\]. *STK11* (OMIM: 604514) functions as a tumor suppressor gene, and the STK11 kinase is involved in the regulation of cell cycle checkpoints \[[@B23]\]. Our findings indicate that the presence of these heterozygous germline variants may have some significance in the mechanism of chromosomal translocation and relapse.

3.2. Analysis of Mutation in Genome Sequencing Data {#sec3.2}
---------------------------------------------------

To identify a somatic mutation associated with relapse, WGS data were generated with an average depth of 30x and sequenced by Illumina HiSeq X-Ten (Illumina, San Diego, CA). We detected 1,009,769 candidate mutations in the relapse sample of the patient (Tables [S1](#supplementary-material-1){ref-type="supplementary-material"}--[S4](#supplementary-material-1){ref-type="supplementary-material"}). We then applied TIDE, and identified 9,631 somatic mutations in the relapse sample of the patient ([Table S5](#supplementary-material-1){ref-type="supplementary-material"}). To investigate potential reasons for relapse after an initial period of remission, we performed WES on a healthy volunteer and the patient before treatment (after the first relapse). TIDE detected 7,815 somatic mutations in the healthy volunteer and 9,643 somatic mutations in the patient before treatment ([Table S6](#supplementary-material-1){ref-type="supplementary-material"}). The somatic mutation (NM_002500.6: c.1142C\>T; \[Gln381∗\]) at chromosome 15 in *MLL* was detected by TIDE in both the relapse sample and in the healthy volunteer. [Figure 1](#fig1){ref-type="fig"} illustrates the mutation in *MLL* in the genome sequencing data of the patient and the healthy volunteer. Therefore, we did not see any significant difference in the number of somatic mutations between the relapse sample and the healthy volunteer. However, we used IGVtools \[[@B24]\] to examine the read counts of the somatic mutation site in MLL. It was difficult to use the IGVtools plot tool to visualize these data because of the high background of the WES data. Therefore, we prepared a histogram of the number of reads of each base, sorted by read position, for the region encompassing the site ([Figure 2](#fig2){ref-type="fig"}). IGVtools can plot this histogram; however, there was not a clear difference in the background between the healthy volunteer and the patient before treatment. The variant calling in the MLL gene was performed by WES analysis and Sanger sequencing and was detected as homozygous in the healthy volunteer. Because we did not find the *MLL* (c.1142C\>T) mutation in the healthy volunteer and only found the heterozygous c.1142C\>T mutation in the healthy volunteer by WES, we concluded that the relapse is likely a coincidence. In addition, we did not find any mutation in *KMT2A* (OMIM: 182117), which is a target of MLL.

In summary, we did not find any strong evidence that the patient had *de novo* somatic mutations after</div></body></html><!-- 2022-06-21 20:53:42 