<!DOCTYPE html><html><head><meta charset='utf-8'><meta name='viewport' content='width=device-width, initial-scale=1'><script async src='https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-8422244967817077' crossorigin=anonymous'></script><link rel='stylesheet' href='/class.css'><script src='/script.js'></script></head><body><div class='topnav'></div><div class='list'></div><div class='stats'><div class='logodiv'><a href='/'><img class='logoimg' src='/img/elephant.svg' /></a></div> </div><div class='nav'><a href='4066.html'> << </a>&nbsp;&nbsp;&nbsp;&nbsp;<a href='4068.html'> >> </a></div><div class='article'>Novel drug resistant mutations: how and why should we detect them?
New mutations develop through the course of natural selection and genetic drift. In the course of the past few years, three novel mutations that confer resistance to an antiretroviral drug, protease inhibitors (PIs), have been found to affect the HIV-1 protease gene, although all isolates were from antiretroviral-na√Øve individuals, suggesting that they must have had selective advantage over their wild-type (WT) counterparts. Two PIs, ritonavir (RTV) and atazanavir (ATV), are boosted with PIs from the beginning of therapy, while lopinavir (LPV) is added only later, which permits the appearance of these viruses. The fact that these three mutants were detected in patients for whom ATV or LPV was not available suggests that other PI resistant mutations also exist that have not yet been characterized. To be able to detect rare mutants, techniques with higher sensitivity are needed. Most of the techniques used to detect minority PIs are based on PCR, i.e. the so called 'Sanger' methods, which only permit detection of a mutation if it is the only mutation in the gene. The main reason is that in this case, a sequence of the protease gene is obtained that can be compared with the sequences of all possible protease gene mutants. To identify minority mutations with 'Sanger' techniques, either a very high sequencing depth or very rare mutants is needed. Some other techniques based on hybrid capture (HC), which use biotinylated RNA probes to detect minority mutants, are more sensitive than 'Sanger' techniques. HC can be considered for general mutation detection and is currently available for detection of minority HIV-1 variants that are resistant to the non-nucleoside reverse transcriptase inhibitors (NNRTIs) and major protease inhibitors (PIs). Recently, a method was reported for the analysis of single polymerase chain reaction (PCR) clones, which includes an additional PCR reaction and HC. This method permits a high sensitivity for minority mutation detection, but it is also very time consuming. Moreover, the cost of sequencing large numbers of clones from the same patient is relatively high. Here we describe a new simple, yet sensitive, molecular method based on single strand conformation polymorphism (SSCP) for detection of minority resistance mutations in HIV-1 genes. The results of our study indicate that SSCP is a very sensitive method, and thus suitable for detection of minority mutants, i.e. 0.1% of HIV-1 in an infected person. In contrast to 'Sanger' techniques and HC, SSCP is not dependent on prior information of what mutations can occur. A novel mutation can be detected in less than 2 h. Because of its rapidity, SSCP is potentially suitable for detection of rare mutations at the time of diagnosis of an infection in HIV-1-infected individuals. This can improve antiretroviral therapy as soon as possible and prevent accumulation of mutations that may lead to drug resistance. The sensitivity and specificity of the method are not affected by the length of protease sequences. In addition, the method allows detection of even minority mutant populations of the two most commonly found mutations that confer LPV and ATV resistance, although these mutations do not cause resistance to all PIs. Thus, SSCP is well suited to determine the frequency of these mutations and to study their behaviour in the course of HIV-1 infection. It also provides additional information when used in combination with other methods, such as HC or 'Sanger' techniques.</div></body></html><!-- 2022-06-25 09:34:32 