<!DOCTYPE html><html><head><meta charset='utf-8'><meta name='viewport' content='width=device-width, initial-scale=1'><script async src='https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-8422244967817077' crossorigin=anonymous'></script><link rel='stylesheet' href='/class.css'><script src='/script.js'></script></head><body><div class='topnav'></div><div class='list'><Br/><a href='4276.html'>Saudi Arabian Crow</a><Br/><a href='4277.html'>Q:

How can I make</a><Br/><a href='4278.html'>Q:

How does the R</a><Br/><a href='4279.html'>Introduction
=====</a><Br/><a href='4280.html'>
Pirates Selling F</a><Br/><a href='4281.html'>Recommendation sys</a><Br/><a href='4282.html'>The invention rela</a><Br/><a href='4283.html'>You should be able</a><Br/><a href='4284.html'>Livinha

Livinha i</a><Br/><a href='4285.html'>What is the common</a></div><div class='stats'><div class='logodiv'><a href='/'><img class='logoimg' src='/img/elephant.svg' /></a></div> <Br/><a href='4274.html'>A phase I/II trial</a><Br/><a href='4273.html'><?xml version="1.0</a><Br/><a href='4272.html'>/*
* Copyright (C)</a><Br/><a href='4271.html'>{
  "name": "react</a><Br/><a href='4270.html'>Facing the Future
</a><Br/><a href='4269.html'>The present invent</a><Br/><a href='4268.html'>A new system, know</a><Br/><a href='4267.html'>The present invent</a><Br/><a href='4266.html'>The present invent</a><Br/><a href='4265.html'>Q:

How can I run </a></div><div class='nav'><a href='4274.html'> << </a>&nbsp;&nbsp;&nbsp;&nbsp;<a href='4276.html'> >> </a></div><div class='article'>Introduction
============

In the field of gene therapy, antisense oligonucleotides are commonly used to downregulate gene expression by selectively binding to the complementary sequence in the mRNA that is crucial for correct gene expression. An oligonucleotide with the ability to efficiently and selectively hybridize to its target mRNA sequence is said to be specific. However, if the duplex that is formed has a large number of mismatches (MMs) relative to the desired binding site sequence (target site), it is termed 'mismatched' or 'non-specific'. Mismatches can be defined as mismatches in the sense strand or the antisense strand, between the target site and the antisense oligonucleotide, with a M mismatch being a mismatch in either strand.

In therapeutic antisense oligonucleotides, a mismatched duplex is more common than a perfect duplex, resulting in greater mismatch binding energy and stronger duplex formation. The presence of MM induces significant thermodynamic and structural changes \[[@b1-ijms-10-02798],[@b2-ijms-10-02798]\] in the mismatched duplex, with MM having been shown to alter the conformational structure of an oligonucleotide, such that the modified oligonucleotide is no longer able to interact with its target site \[[@b3-ijms-10-02798]--[@b5-ijms-10-02798]\].

A major obstacle to *in vivo* gene silencing using antisense technology is that while antisense oligonucleotides are specific in a buffer solution, they do not bind to the intended target site in the cell nucleus \[[@b6-ijms-10-02798],[@b7-ijms-10-02798]\]. We observed the same phenomenon in a mismatched duplex, *i.e.*, a mismatch (MM)-modified oligonucleotide duplex bound less efficiently to the target site as compared to the completely complementary duplex. This study aimed to determine the binding patterns and thermodynamic parameters of a mismatched duplex and provide insights for the future development of highly specific, mismatched, and mismatch-free antisense oligonucleotides for therapeutics.

2.. Results and Discussion
==========================

2.1.. Oligonucleotide duplexes
------------------------------

The sequences and structures of the oligonucleotides used in this study were obtained from Oligonucleotide database, Mfold, and the Vienna RNA package \[[@b8-ijms-10-02798]--[@b10-ijms-10-02798]\]. Mismatches were created using a base pair in which a methylene (methylene group with a hydrogen atom attached) replaced a carbon atom (C~2~H~4~) of the base, which allowed us to avoid steric problems with chemical changes in the side chain of the nucleobases, such as replacement of a methyl group by an alkyl group. This led to the creation of more standard and well defined bases for use as C versus T bases. For example, the guanine (G) base was changed to xanthine (X), and the cytosine (C) base was changed to oxanine (O). A single mismatch was created between T14 and X14, T15 and X15, C16 and X16, C17 and O17, and between G19 and O19 ([Figure 1A](#f1-ijms-10-02798){ref-type="fig"}). The complete sequences of the oligonucleotides used in this study are given in [Table 1](#t1-ijms-10-02798){ref-type="table"}.

The duplexes used in this study were fully matched or totally mismatched, that is, the sequence of one strand in the duplex contained the complete complement of the other strand. The structure and energetics of an oligonucleotide and its Watson-Crick base pair are also well known \[[@b11-ijms-10-02798],[@b12-ijms-10-02798]\]. In this study, X, O, C, and G were used as a Watson-Crick pair and T was used as a thymidine.

2.2.. Thermodynamic analysis of a mismatched duplex
---------------------------------------------------

Duplex formation is stabilized by favorable stacking interactions of neighboring nucleotides, which are primarily determined by the free energy, ∆G^0^, of a single base pair (G-C/C-G: 0 kcal/mol; A-T/T-A: −1.6 kcal/mol; C-G/G-C: −1.8 kcal/mol) \[[@b13-ijms-10-02798]\]. When comparing two oligonucleotides, the duplex stabilities of two complementary oligonucleotides are usually close to each other due to a similar number of base pairs. On the other hand, non-complementary oligonucleotide duplexes do not exhibit the same level of stability and flexibility that is typical of those formed between complementary oligonucleotides. This mismatch effect was reported in several studies, for example, to create and compare mismatched complexes between two molecules; a single nucleotide mismatch (A-A) in the DNA double helix \[[@b14-ijms-10-02798]\] was used as a model of mismatch in gene therapy, and the effect of the thymine dimer, on a double-stranded DNA \[[@b15-ijms-10-02798]\]. In some cases, a specific duplex was found to be more stable than a fully mismatched duplex with an all-T mismatch. This result is believed to be the result of a higher binding energy of a specific pair in the duplex with all T mismatches compared to the free energy of binding of a mismatch-free double helix \[[@b16-ijms-10-02798],[@b17-ijms-10-02798]\]. As can be seen from the results obtained in this study, mismatched duplexes do have lower thermodynamic stabilities than the corresponding fully complementary duplexes, although mismatched duplexes were created with a thymine, which has the lowest level of stability, which makes mismatches more likely to occur than in other positions. It should also be noted that mismatched duplexes are much less stable in the presence of monovalent salt than are fully complementary duplexes. This is probably due to the loss of two electrostatic charges.

2.3.. Temperature dependence of UV melting
------------------------------------------

To determine the binding energy of a duplex structure, a UV melting experiment was conducted on the samples used in the thermodynamic analysis. The molar absorption coefficient of each oligonucleotide and the UV melting curves for the duplexes are shown in [Figure 2](#f2-ijms-10-02798){ref-type="fig"}. The absorbance at 260 nm is a standard and widely accepted means to estimate melting points. Changes in absorbance at 260 nm and 305 nm have been used as a means of detecting the dissociation of duplexes \[[@b18-ijms-10-02798]\]. Generally, the melting temperatures (*T~m~*) and melting slopes for fully complementary oligonucleotides are similar to each other and do not change upon formation of a duplex with a mismatch. The *T~m~* values for the fully matched duplexes, G19-O19 and C16-O16, were 53.25 °C and 59.4 °C, respectively. The *T~m~* values of the mismatched duplexes are slightly lower than those for the fully matched duplexes, as shown in [Table 2](#t2-ijms-10-02798){ref-type="table"}. The thermodynamic data from the melting profiles are shown in [Table 3](#t3-ijms-10-02798){ref-type="table"}. The thermodynamic stability, as determined by the ∆G^0^ of the single base pair, is very small for the mismatched duplexes, except for C17-O17, which is lower than that for C16-O16. These results are consistent with other studies that used the same type of base as the mismatched pair, thymine \[[@b19-ijms-10-02798],[@b20-ijms-10-02798]\].

2.4.. Thermodynamic parameters of DNA duplex formation
------------------------------------------------------

The enthalpies (∆H°) of all duplexes are small, ranging from 6.07 kcal/mol to 6.35 kcal/mol, showing that the oligonucleotides bind to their targets in a typical fashion. The entropies (∆S°) of all duplexes are also low, ranging from −21.56 cal/mol·°C to −21.83 cal/mol·°C, showing that the change in enthalpy is large and the change in entropy is small. Thus, the entropy and enthalpy are important factors for understanding the structure of nucleic acids \[[@b21-ijms-10-02798]--[@b23-ijms-10-02798]\]. As shown in [Table 3](#t3-ijms-10-02798){ref-type="table"}, in the presence of T or G, the ∆G^0^ of a mismatch is greater than that for a mismatched complex. This result is due to the large enthalpic and entropic terms associated with a single base pair that cannot be compensated by favorable enthalpic and entropic contributions due to favorable stacking interactions. The MM effect on T14 in G19-O19, and on G16 and C17 in T16-G15 were favorable, which may be attributed to the greater number of stacking interactions due to the increased number of available binding sites, making them much more favorable, as shown in [Figure 3](#f3-ijms-10-02798){ref-type="fig"}. The stabilities of T16-G15 and T14-C17 in G19-O19 were higher than that of G15-C17 in C16-O16</div></body></html><!-- 2022-06-25 09:55:46 